e2e_playwright/st_dataframe_interactions_test.py

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
I'm slightly worried about the positions that are hardcoded here especially if dataframe changes occur or just flakiness. I can't think of a better way to resolve this so I guess I'm ok with this for now but we should definitely try to monitor these...